### PR TITLE
EPMRPP-34391: tag shows long launch name now

### PR DESCRIPTION
**Pull request by csigataraj**
fixed long name bug

### CODE REVIEW

File: app/src/pages/inside/common/selectedItems/selectedItem/selectedItem.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .selected-item {
-  height: 20px;
+  // height: 20px;
```
Comment (AmsterGet) [inline app/src/pages/inside/common/selectedItems/selectedItem/selectedItem.scss]: Please, remove unnecessary comments

Comment (csigataraj) [reply app/src/pages/inside/common/selectedItems/selectedItem/selectedItem.scss]: removed

File: app/src/pages/inside/common/selectedItems/selectedItems.scss [hunk 2]
```diff
@@ -3,6 +3,8 @@
   padding-top: 15px;
   padding-left: 15px;
   padding-right: 15px;
+  word-break: break-all;
+  word-break: break-word;
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/selectedItems/selectedItems.scss]: Let's use `@supports` directive to make sure every browser will use correct style:
```
  word-break: break-all;

  @supports (word-break: break-word) {
    word-break: break-word
  }
```

File: app/src/pages/inside/common/selectedItems/selectedItem/selectedItem.scss [hunk 3]
```diff
@@ -1,5 +1,4 @@
 .selected-item {
-  height: 20px;
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/selectedItems/selectedItem/selectedItem.scss]: Maybe it's better to leave this as `min-height`?

Comment (csigataraj) [reply app/src/pages/inside/common/selectedItems/selectedItem/selectedItem.scss]: have put it back as min-height

File: app/src/pages/inside/common/selectedItems/selectedItems.scss (hunk 2 above)
Comment (csigataraj) [reply app/src/pages/inside/common/selectedItems/selectedItems.scss]: replaced it as suggested

File: app/src/pages/inside/common/selectedItems/selectedItems.scss [hunk 4]
```diff
@@ -3,6 +3,11 @@
   padding-top: 15px;
   padding-left: 15px;
   padding-right: 15px;
+  word-break: break-all;
+
+  @supports (word-break: break-word) {
+    word-break: break-word
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/selectedItems/selectedItems.scss]: Please add missing semicolon

Comment (csigataraj) [reply app/src/pages/inside/common/selectedItems/selectedItems.scss]: added

**Pull request merged**